.circleci/config.yml

```diff
@@ -21,7 +21,9 @@ jobs:
       - node/test:
           post-node-js-install-steps:
             - run: test/ci/before_install.sh
+          override-ci-command: yarn install --frozen-lockfile --ignore-engines
```

Comment:
question: what dependency requires us to add `--ignore-engines`?

Reply:
In my case, `ts-api-utils` via `@typescript-eslint#eslint-plugin`, see [this failed run](https://app.circleci.com/pipelines/github/electron/electron-packager/1054/workflows/da09ed73-6f94-4b00-83cc-f3eaf623fe44/jobs/5885?invite=true#step-113-204_120). I actually copied this line from the [CFA releases PR](https://github.com/electron/electron-packager/pull/1566/commits/332ae93f798d97b0612599fef974ac8f2fd357a6#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R25), I imagine it was included there for the same reason.